test/unit_tests/sundomeigest/CMakeLists.txt

```
@@ -0,0 +1,52 @@
+# ---------------------------------------------------------------
+# Programmer(s): Mustafa Aggul @ SMU
+# ---------------------------------------------------------------
+# SUNDIALS Copyright Start
+# Copyright (c) 2002-2025, Lawrence Livermore National Security
+# and Southern Methodist University.
+# All rights reserved.
+#
+# See the top-level LICENSE and NOTICE files for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+# SUNDIALS Copyright End
+# ---------------------------------------------------------------
+# test/unit_tests/sundomeigest level CMakeLists.txt for SUNDIALS
+# ---------------------------------------------------------------
+
+if(ENABLE_CALIPER AND SUNDIALS_BUILD_WITH_PROFILING)
+  include_directories(${caliper_INCLUDE_DIR})
+  set(EXE_EXTRA_LINK_LIBS ${EXE_EXTRA_LINK_LIBS} caliper)
+endif()
+
+set(EXE_EXTRA_LINK_LIBS ${EXE_EXTRA_LINK_LIBS} sundials_sundomeigestpower)
+
+if(ENABLE_LAPACK)
+  set(EXE_EXTRA_LINK_LIBS ${EXE_EXTRA_LINK_LIBS} sundials_sundomeigestarnoldi)
+endif()
+
+# Always add the power iteration examples
+add_subdirectory(Power)
+
+# Build the sundials_sundomeigest test utilities
+add_library(test_sundomeigest_obj OBJECT test_sundomeigest.c)
```

List the header also
```suggestion
add_library(test_sundomeigest_obj OBJECT test_sundomeigest.c test_sundomeeigest.h)
```